sources/api/schnauzer/Cargo.toml

```diff
@@ -19,14 +19,18 @@ handlebars = "4"
 http = "0.2"
 lazy_static = "1"
 log = "0.4"
+maplit = "1.0"
```

Comment:
Perhaps I missed it being used outside of tests, but if `maplit` is only used in tests, can we move it to `[dev-dependencies]` section?